ksqldb-execution/src/main/java/io/confluent/ksql/execution/plan/StreamStreamJoin.java

```diff
@@ -137,11 +137,11 @@ public ColumnName getKeyColName() {
   }
 
   public Duration getAfterMillis() {
-    return afterMillis;
+    return Duration.ofMillis(afterMillis.toMillis());
```

Comment:
Make deep-copy. We know we only care about millisecond precision, so it should be fine.